tor/core/inbox.py

```diff
@@ -112,9 +112,13 @@ def check_inbox(config):
                 process_override(reply, config)
                 reply.mark_read()
                 return
+
             if 'good bot' in reply.body.lower() or 'bad bot' in reply.body.lower():
                 # please stop emailing me, I just don't care
                 reply.mark_read()
+                return
+
+            send_to_slack('Unknown reply: ' + reply.body, config)
```

Comment:
Can we also include the author (if there is one)? Example:

`send_to_slack('Unknown reply by {author}: {body}'.format(author=reply.author, body=reply.body), config)`